docs/registry.md

```diff
@@ -6,15 +6,15 @@ With containerd, `docker.io` is the default image registry. You can also set up
 
 To configure image registries create/modify the `/etc/containerd/config.toml` as follows:
 ```toml
```

Comment:
Can we add `version = 2` header here?

Reply:
Also, we should add a note about the difference between v1 and v2

Reply:
Agree

Reply:
addressed in 51913a5dc3a4c7398e00e5241657a4b6a440da0d, please check